activejob/test/cases/test_helper_test.rb

```diff
@@ -521,3 +522,9 @@ def test_assert_job_has_custom_queue_adapter_set
     assert_instance_of CustomQueueAdapter, HelloJob.queue_adapter
   end
 end
+
+class InheritedJobTest < ActiveJob::TestCase
+  def test_queue_adapter_is_test_adapter
+    assert_instance_of ActiveJob::QueueAdapters::TestAdapter, InheritedJob.queue_adapter
```

Comment:
Ahh, understand now - it's testing that the adapter is overridden. Sorry - looks good 👍